doc/design/embargo-mvp.md

```diff
@@ -0,0 +1,141 @@
+# Embargo Minimum Viable Product
+Due to time constraints, we are implementing the core functionality for embargoing dandisets ASAP.
+This document is dedicated to the minimum viable product.
+A subsequent document will describe the full suite of features.
+
+
+# User experience
+
+## Web
+When creating a new dandiset, users will have checkbox to choose whether or not the dandiset should be embargoed.
+If so, the user must also specify an NIH award number (https://era.nih.gov/files/Deciphering_NIH_Application.pdf).
+The award number is an optional argument that is passed to the create dandiset endpoint.
+The award number will be added as a new `Funder` on an automatically generated Contributor Organization in the metadata.
+
+Instead of the normal `Publish` button and version browser, the DLP will instead show a `Release` button.
+Clicking it will open a confirmation modal informing the user that they are unembargoing, all their data will be publicized, there is no undoing, and that it can take some time for large dandisets.
+Confirming will lock the dandiset for the duration of the release.
+Once the release finishes, the dandiset will be like any other unpublished draft dandiset.
+
+Once created, an embargoed dandiset is only visible or searchable to owners.
+Instead of the `draft` chip, an `embargoed` chip should be used on the dandiset listing pages.
+These dandisets+chips will only appear in the listings to owners of the embargoed dandisets.
+
+## CLI
+The CLI experience will be basically unchanged.
+Uploads to an embargoed dandiset will function exactly the same from an API perspective, the data simply goes to a different location.
+
+### TODO: how to browse/download embargoed zarr files?
+Credentials need to be generated, distributed, and used by the client.
```

Comment:
Is this going to be part of the MVP?

Reply:
i think credentials are part of mvp almost independent of zarr, but essential for zarr. since people should be able to access the endpoint without always going through pre-signed urls right? for example to load an hdf5 file via streaming one would need the s3 path. a presigned url doesn't work for s3 read access for all tools.